### DISCUSSION TITLE
language pack

### DISCUSSION THREAD

**Discussion by TuShanShengYao** [AutoDarkMode/Windows-Auto-Night-Mode#712] [Q&A] [answered]
这个软件没有中文语言包吗？
Does this software not have a Chinese language pack?

**Comment by Spiritreader** [comment, marked as answer]
Yes, we both have traditional and simplified
![image](https://user-images.githubusercontent.com/18438899/232581268-273b8535-520a-4242-a164-2e5d1c45bbeb.png)
